File: Atributos de validación Data Annotations multiidioma/[VB]-Atributos de validación Data Annotations multiidioma/VB/LocalizeDataAnnotations/Models/Persona.vb
Imports System.ComponentModel
Imports System.ComponentModel.DataAnnotations

Public Class Persona
    Implements IPersona

    <Required(ErrorMessage:="Debe introducir un nombre")> _
    <DisplayName("Nombre")> _
    Public Property Nombre As String Implements IPersona.Nombre

    <DisplayName("Apellido")> _
    Public Property Apellido As String Implements IPersona.Apellido

    <DisplayName("Ciudad")> _
    <StringLength(20, MinimumLength:=4, ErrorMessage:="El nombre de Ciudad debe tener una longitud entre 4 y 20 caracteres")> _
    Public Property Ciudad As String Implements IPersona.Ciudad

    <DisplayName("Correo electrónico")> _
    Public Property Email As String Implements IPersona.Email

End Class

File: Atributos de validación Data Annotations multiidioma/[C#]-Atributos de validación Data Annotations multiidioma/C#/LocalizeDataAnnotations/Models/Persona.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace LocalizeDataAnnotations.Models
{
    public class Persona: IPersona
    {

        [Required(ErrorMessage = "Debe introducir un nombre")]
        [DisplayName("Nombre")]
        public string Nombre { get; set; }

        [DisplayName("Apellido")]
        public string Apellido { get; set; }

        [DisplayName("Ciudad")]
        [StringLength(20, MinimumLength=4, ErrorMessage="El nombre de Ciudad debe tener una longitud entre 4 y 20 caracteres")]
        public string Ciudad { get; set; }

        [DisplayName("Correo electrónico")]
        public string Email { get; set; }

    }
}